sdk/formrecognizer/Azure.AI.FormRecognizer/tests/samples/Sample7_ManageCustomModelsAsync.cs

```diff
@@ -35,8 +35,8 @@ await foreach (CustomFormModelInfo modelInfo in models)
                 Console.WriteLine($"Custom Model Info:");
                 Console.WriteLine($"    Model Id: {modelInfo.ModelId}");
                 Console.WriteLine($"    Model Status: {modelInfo.Status}");
-                Console.WriteLine($"    Requested on: {modelInfo.RequestedOn}");
-                Console.WriteLine($"    Completed on: : {modelInfo.CompletedOn}");
+                Console.WriteLine($"    Training model started on: {modelInfo.TrainingStartedOn}");
+                Console.WriteLine($"    Training model completed on: : {modelInfo.TrainingCompletedOn}");
```

Comment:
Does the README need to be updated like this too?

Reply:
_plop_ yes! Good catch. Will update